src/dbi.rs

```diff
@@ -364,25 +373,25 @@ impl From<u16> for MachineType {
 /// `struct SC` in Microsoft's code:
 /// https://github.com/Microsoft/microsoft-pdb/blob/082c5290e5aff028ae84e43affa8be717aa7af73/PDB/include/dbicommon.h#L42
 #[derive(Debug, Copy, Clone)]
-pub(crate) struct DBISectionContribution {
+pub struct DBISectionContribution {
     /// The index of the section.
-    section: u16,
+    pub section: u16,
     _padding1: u16,
```

Comment:
Could you remove the padding attributes so that this becomes a fully public `struct`? That way, we're able to construct this type in the future (in case we add a write option, etc.)